esphome/exports.sh

```diff
@@ -0,0 +1,2 @@
+export APP_ESPHOME_PORT="8093"
```

Comment:
Sorry @ShonP40 just one last thing, ESPHome dashboard is served on it's default port of `6052` in the container but we map that to port `8093` on the host. Is there a reason for this? No other app is using `6052` so it's available, shall we use `6052` on both the container and host so the user accesses ESPHome on it's default port port?